src/hdusd/export/object.py

```diff
@@ -25,6 +25,7 @@
 
 
 SUPPORTED_TYPES = ('MESH', 'LIGHT', 'CURVE', 'FONT', 'SURFACE', 'META', 'CAMERA', 'EMPTY')
+USD_CAMERA = "USD Camera"
```

Comment:
Better move USD_CAMERA constant to usd_collection.py

Reply:
I found it causes a lot of  'circular import' issues.